app/views/admin_mailer/linkedin_weekly_profiles.html.erb

```diff
@@ -0,0 +1,26 @@
+<p>Here are the developer with linkedin profiles updated in past 7 days:</p>
```

Comment:
Can you add a mailer preview? I'm having a hard time wrapping my head around what this will look like.